src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's contact in the address book.
+ * Is a superclass of Phone, Email and Address classes
+ */
+public class Contact {
+
+    public String value;
+    protected boolean isPrivate;
```

Comment:
It's a good practice to limit all class members permission whenever possible.
Consider setting permission to private